externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/udf/KDFRegistry.scala

```diff
@@ -36,6 +36,13 @@ object KDFRegistry {
     "string",
     "1.3.0")
 
+  val system_user: KyuubiDefinedFunction = create(
+    "system_user",
+    udf(() => System.getProperty("user.name")).asNonNullable(),
```

Comment:
should we use a lazy val for `System.getProperty("user.name")` and return it directly? @yaooqinn